### DISCUSSION TITLE
Documentation from the future

### DISCUSSION THREAD

**Discussion by bphunter1972** [Textualize/textual#2309] [Announcements]
[Current documentation](https://textual.textualize.io/widgets/data_table/#textual.widgets._data_table.DataTable.remove_row) as of this writing shows the DataTable.remove_row() method. 

However, this method was added in #2253 and was not in the 0.19.1 release (#2247).

This indicates to me that the documentation is being generated from the main trunk and not from the release point. I would guess that's not intentional, since it directed me to a function I don't have.

**Comment by willmcgugan** [comment]
That's right. We still need to figure out how to version the docs. Next version is a day or two away.
